### ISSUE TITLE
Data Point - Unable to Add New 'Saved Posts' past ID: 0

### ISSUE THREAD

**Issue by CMDRLucienn** [DBnR1/EDTB#10]
Unable to add a second Saved Post in Data Point page; returns the following error:

```
Error

Duplicate entry '0' for key 'PRIMARY'
```

The form for 'Add Record' contains an auto-increment field for ID (primary key), so it is not possible to change this value from 0, which I imagine is causing the error (failing to auto-increment).

**Comment by DBnR1** [comment]
I'm unable to replicate this issue on my end. 
Can you confirm you're using either version 1.2.2 of ED ToolBox or version 1.2.3 (released today)?

**Comment by CMDRLucienn** [comment]
I was running 1.2.2 when the error occurred, and I upgraded to 1.2.3 just now to see if the issue persists. It still seems to be occurring.

I've attached four images below depicting my version number as well as the error message.

http://prntscr.com/amyr2u - version number
http://prntscr.com/amyscr - Page before addition
http://prntscr.com/amystk - attempted addition
http://prntscr.com/amyt0w - Error message

**Comment by CMDRLucienn** [comment]
Just as a test, I entered the Adminer interface and deleted Record ID 0 from the table, and resubmitted - the first record enters fine, and again additional records fail to auto-increment.

**Comment by CMDRLucienn** [comment]
I poked around in the Adminer interface; the radio button for 'Auto Increment' in x_saved_posts table was not selected. After selection, the error appears to have fixed itself:

http://prntscr.com/amyyik - Radio button

I think its possible that this could be due to repeated database migrations. I started using EDTB pre-1.0.0, and the bug may have carried over from back then through database upgrades, rather than be a bug in the current builds.
